docs/workflow/blocks/mapping.rst

```diff
@@ -600,15 +600,41 @@ Examples:
 21. json
 --------
 
-Converts a value to a JSON string.
+Converts a value to a JSON string using [`json-stringify-safe`](https://www.npmjs.com/package/json-stringify-safe) behind the scenes for safe stringification.
```

Comment:
It would be better to say what it does up front so people don't have to read too far, and if you want `json-stringify-safe` included still, whose unique features it doesn't use, something like this would be better:
```suggestion
Converts a value to a JSON string, which can optionally be indented with a number of spaces for `pretty printing`. It works much like `JSON.stringify`.  
Internally, [`json-stringify-safe`](https://www.npmjs.com/package/json-stringify-safe) is used to keep dependencies consistent and avoid introducing another stringifier; cycles are not expected in normal flows.
```

Reply:
Changes agreed during the meeting with @lukestanley 

- Optional arguments can be provided to prittify the output
- It is using [`json-stringify-safe`](https://www.npmjs.com/package/json-stringify-safe) behind the scenes